index.html

```diff
@@ -3,12 +3,16 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>AJAX Project</title>
-  <link rel="icon" href="images/favicon.ico">
+  <meta property="title" content="Filmster">
+  <meta property="description" content="A web application for movie enthusiats who want to look up detailed information about a movie.">
+  <meta property="image" content="images/logo.ico">
+  <meta property="url" content="https://devjosephlee.github.io/Filmster/">
```

Comment:
The values of the `property` attributes here are incorrect:
![image](https://user-images.githubusercontent.com/32880187/142485803-aaddcfb3-d270-4a40-a5bd-b7865d578a1c.png)